### PR TITLE
A Ping Pong game added html js and css files

### PR DESCRIPTION
**Pull request by SahilRout**
A ping pong game made using html CSS and Js added those to there respective folders 

Thank You

### CODE REVIEW

**Comment by NOiR-07** [comment]
@SahilRout Provide the screenshots of your projects for overview

**Comment by SahilRout** [comment]
> @SahilRout Provide the screenshots of your projects for overview

here it i
Screenshot 2021-10-04 194155
s

File: Public/pong-game.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
```
Comment (NOiR-07) [inline Public/pong-game.html:8]: Change the href of css, make it as . ./assets/css/foldername

File: Public/pong-game.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        <div class="bat" id="bat1"></div>
+        <div class="bat" id="bat2"></div>
+        <div class="player1Score">0</div>
+        <div class="player2Score">0</div>
+        <h1 class="message">
+            Press Enter To Play Pong
+        </h1>
+    </div>
+</body>
+<script src="script.js"></script>
```
Comment (NOiR-07) [inline Public/pong-game.html:27]: Same for the js file

**Pull request closed without merging**